languages/auto-tags-smart-uk.po

```diff
@@ -0,0 +1,142 @@
+# Translation file for Auto Tags Smart
+# Copyright (C) 2025 Maison da Silva
+# This file is distributed under the same license as the Auto Tags Smart package.
+msgid ""
+msgstr ""
+"Project-Id-Version: Auto Tags Smart 1.0\n"
+"Report-Msgid-Bugs-To: https://maisondasilva.com.br/\n"
+"POT-Creation-Date: 2025-07-21 12:28+0000\n"
+"PO-Revision-Date: 2025-07-24 09:03+0000\n"
+"Last-Translator: Eugene Fox <eugene@xfox111.net>\n"
+"Language-Team: Ukrainian (Ukraine)\n"
+"Language: uk\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#~ msgid "Auto Tags Smart"
+#~ msgstr "Розумні Авто-позначки"
+
+#~ msgid "Settings"
+#~ msgstr "Налаштування"
+
+#: auto-tags-smart.php:29
+msgid "Select All"
+msgstr "Вибрати все"
+
+#: auto-tags-smart.php:30
+msgid "Deselect All"
+msgstr "Зняти виділення"
+
+#: auto-tags-smart.php:31
+msgid "STATUS: Active ✓"
+msgstr "СТАТУС: Активно ✓"
+
+#: auto-tags-smart.php:32
+msgid "STATUS: Active but nothing to examine"
+msgstr "СТАТУС: Активно, немає доступних записів для аналізу"
+
+#: auto-tags-smart.php:33
+msgid "STATUS: Inactive"
+msgstr "СТАТУС: Неактивно"
+
+#: admin/auto-tags-smart-admin.php:41
+msgid "Enable Auto Tags Smart"
+msgstr "Увімкнути розумні авто-позначки"
+
+#: admin/auto-tags-smart-admin.php:42
+msgid "Enables automatic detection of existing tags in posts."
+msgstr "Увімкнути автоматичне визначення існуючих позначкiв у записах."
+
+#: admin/auto-tags-smart-admin.php:49
+msgid "Block manually added tags"
+msgstr "Видаляти користувацькі позначки"
+
+#: admin/auto-tags-smart-admin.php:50
+msgid "Removes manually added tags and uses only automatically detected ones."
+msgstr "Коли увімкнено, опція видаляє вручну додані позначки та використовує лише автоматично визначені."
+
+#: admin/auto-tags-smart-admin.php:49
+msgid "Examine post title"
+msgstr "Аналізувати заголовки записів"
+
+#: admin/auto-tags-smart-admin.php:50
+msgid "Analyzes the post title to find tags."
+msgstr "Шукати позначки у заголовках записів."
+
+#: admin/auto-tags-smart-admin.php:55
+msgid "Examine post content"
+msgstr "Аналізувати вміст записів"
+
+#: admin/auto-tags-smart-admin.php:56
+msgid "Analyzes the post content to find tags."
+msgstr "Шукати позначки у тексті записів."
+
+#: admin/auto-tags-smart-admin.php:62
+msgid "Filter by category"
+msgstr "Фільтрувати за категоріями"
+
+#: admin/auto-tags-smart-admin.php:63
+msgid "Only applies tags to posts in selected categories."
+msgstr "Застосовувати позначки лише до вибраних категорій."
+
+#: admin/auto-tags-smart-admin.php:68
+msgid "Tag Length"
+msgstr "Довжина позначкiв"
+
+#: admin/auto-tags-smart-admin.php:73
+msgid "Minimum tag length"
+msgstr "Мінімальна довжина позначкiв:"

```

Comment:
Looks like this string is no longer being used, since you replaced it with `Tag Length`